### PR TITLE
ENDOC-327 add base for 6.3.2 docs

### PR DESCRIPTION
**Pull request by joewhite101**
This copies next to 6.3.2 and makes it the default.

### CODE REVIEW

File: vuepress/docs/.vuepress/plugins/entando-nav-version/enhanceAppFile.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ export default ({ router, isServer }) => {
         }
         //Reuse the versioned part of the path, either (/prefix)/vX.Y or (/prefix)/next
         var start = (versionPos >= 0) ? versionPos : nextPos;
-        var pos = pathname.indexOf("/", start + 1);
+        var pos = pathname.indexOf("/", start + 2);
```
Comment (nshaw) [inline vuepress/docs/.vuepress/plugins/entando-nav-version/enhanceAppFile.js:18]: @joewhite101 , this seems off to me.

**Review by nshaw — APPROVED** [review]
Just the one concern re the nav.

File: vuepress/docs/.vuepress/plugins/entando-nav-version/enhanceAppFile.js (hunk 1 above)
Comment (joewhite101) [reply vuepress/docs/.vuepress/plugins/entando-nav-version/enhanceAppFile.js:18]: @nshaw You might be right. this change was required in order to reflect the 6.3.2 version in the header in the dropdown of the left nav. Without it when you pick 6.3.2 from the list it says "Entando 6.3 Tutorials" Entando 6.3 Docs" even though you're on the 6.3.2 path.

**Comment by nshaw** [comment]
@joewhite101 , this still has the warnings on the top-level docs and tutorials pages. I think you were planning to work through the release steps still from the README but just calling it out.

**Comment by joewhite101** [comment]
> @joewhite101 , this still has the warnings on the top-level docs and tutorials pages. I think you were planning to work through the release steps still from the README but just calling it out.

Good call. Removed so we don't forget it later.

**Pull request merged**